sourcing-new/src/main/scala/ch/epfl/bluebrain/nexus/sourcingnew/SourcingModule.scala

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.bluebrain.nexus.sourcingnew
+
+import akka.actor.typed.ActorSystem
+import akka.stream.alpakka.cassandra.scaladsl.CassandraSession
+import cats.effect.{Async, ContextShift}
+import ch.epfl.bluebrain.nexus.sourcingnew.EventLog.{CassandraEventLogFactory, JdbcEventLogFactory}
+import ch.epfl.bluebrain.nexus.sourcingnew.projections.cassandra.{CassandraConfig, CassandraProjection, CassandraSchemaMigration, ProjectionConfig}
+import ch.epfl.bluebrain.nexus.sourcingnew.projections.jdbc.{JdbcConfig, JdbcProjection, JdbcSchemaMigration}
+import ch.epfl.bluebrain.nexus.sourcingnew.projections.{Projection, SchemaMigration}
+import distage.{Axis, ModuleDef, Tag, TagK}
+import doobie.util.transactor.Transactor
+import io.circe.{Decoder, Encoder}
+
+object Persistence extends Axis {
```

Comment:
drop file and port everything to monix-bio

Reply:
Yep, that is one of the consequences of our decisions last week